steps/terraform.yaml

```diff
@@ -40,6 +40,9 @@ parameters:
   - name: serviceConnection
     default: ''
 
+  - name: networkServiceConnection
```

Comment:
this is an implementation detail, can't the template in the repository handle this in some way and not have to pass it down here?